app/services/prior_authority/assessment_syncer.rb

```diff
@@ -5,9 +5,13 @@ def call(application)
         app_store_record = AppStoreClient.new.get(application.id)
 
         sync_overall_comment(application, app_store_record)
```

Comment:
So Joel [raised the point](https://mojdt.slack.com/archives/C05212WRK5J/p1711041737323699?thread_ts=1711041195.429809&cid=C05212WRK5J) that this method should be called for `rejected`/`granted` and `part_grant` applications, but _not_ for `sent_back` applications, because (a) when an application is sent back there will not be a single overall comment, and (b) even if there was, this method assumes that there will be an `Event::Decision` event. (I think it's the absence of that that is making your specs fail)